modules/virt-schedule-best-match-cpu-model-vms.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-scheduling-vms.adoc
+
+[id="virt-schedule-best-match-cpu-model-vms_{context}"]
+= Scheduling virtual machines with the best match for CPU model
+
+When a CPU model is not defined for a node and a virtual machine (VM), the CPU model for the VM is set to `host-model` and the VM is scheduled on the node that offers the best match for the VM's CPU model.
+
+.Procedure
+
+. Edit the `domain` spec of your VM configuration file. The following example shows `host-model` being specified for the Virtual Machine Instance (VMI):
++
+[source,yaml]
+----
+metadata:
```

Comment:
This example is missing `kind` and `apiVersion` and the indentation is wrong, the `cpu` field should be inside the `domain` field